src/modules/emails/emails.controller.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  BadRequestException,
+  Body,
+  Controller,
+  ForbiddenException,
+  Headers,
+  HttpStatus,
+  InternalServerErrorException,
+  ParseArrayPipe,
+  Post,
+  UnauthorizedException,
+  UnprocessableEntityException,
+} from '@nestjs/common';
+import { ApiBadRequestResponse, ApiBody, ApiCreatedResponse, ApiOperation, ApiTags, ApiUnprocessableEntityResponse } from '@nestjs/swagger';
+import { PostEmailsResponseDto } from '@ukef/helper-modules/govuk-notify/dto/post-emails-response.dto';
+
+import { PostEmailsRequestItemDto } from './dto/post-emails-request.dto';
+import { EmailsService } from './emails.service';
+
+@ApiTags('emails')
+@Controller('emails')
+export class EmailsController {
+  constructor(private readonly emailsService: EmailsService) {}
+
+  @Post()
+  @ApiOperation({
+    summary: 'Send email using to GOV.UK Notify service',
+  })
+  @ApiBody({ type: [PostEmailsRequestItemDto] })
+  @ApiCreatedResponse({
+    status: HttpStatus.CREATED,
+    description: 'Returns information about email transaction.',
+    type: [PostEmailsResponseDto],
+  })
+  @ApiBadRequestResponse({
+    description: 'Bad request',
+  })
+  @ApiUnprocessableEntityResponse({
+    description: 'No GOV.UK Notify response',
+  })
+  /**
+   * Verify request and send email
+   * @param {String} govUkNotifyKey
+   * @param {PostEmailsRequestItemDto[]]} post email request
+   *
+   * @returns {Promise.<PostEmailsResponseDto>} GOV.UK Notify response
+   *
+   * @throws {BadRequestException}
+   * @throws {UnauthorizedException}
+   * @throws {ForbiddenException}
+   * @throws {Error}
+   * @throws {UnprocessableEntityException}
+   * @throws {InternalServerErrorException}
+   */
+  postEmail(
+    @Headers('govUkNotifyKey') govUkNotifyKey: string,
```

Comment:
Why do we send the `govUkNotifyKey` in the request from DTFS, rather than it being an env var in MDM?

Reply:
MDM API will be used by different applications and they will have their own `keys`. This was part of requirements in the user story.